remoteappmanager/db/orm.py

```diff
@@ -1,8 +1,10 @@
 import contextlib
+import sqlite3
```

Comment:
but not necessarily compiled in. You can disable it (at least it was optional in py27, I don't know if it is in py34)